server/config/routes.rb

```diff
@@ -23,4 +23,7 @@
   end
 
   resources :categories, :comments
+  get 'adverts/:id/liked', to: 'adverts#liked' 
+  post 'adverts/:id/likes', to: 'likes#create'
+  delete 'adverts/:id/likes', to: 'likes#destroy'
 end
```

Comment:
would be good to use nested resources here, similar to comments...
actually those 3 routes had to be added at line 23, e.g.:

```ruby
resources :adverts do
    resources :comments
    resources :likes, only: [:index, :create, :destroy]
end
```

and action `liked` should be moved into likes controller and be named index according to REST :)